app/grandchallenge/core/static/css/core.css

```diff
@@ -128,7 +128,7 @@ div.sidebar h1 {
 /* Little more spacing for checkbox */
 
 input[type=checkbox] {
-    margin-top: .3rem !important;
+    margin-right: .3rem !important;
```

Comment:
This also affects the checkboxes on forms, you should check those (log in as demo and then so to challenge settings)